packages/wp-now/README.md

```diff
@@ -4,27 +4,109 @@
 
 ## Getting Started
 
+Before getting started, make sure you have `nvm` installed. If you need to install it first,
```

Comment:
It's okay to require this for folks cloning the repo. Let's just add a note that it's not actually needed if you install `wp-now` from npm.